modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -795,15 +804,54 @@ Ransomware Protection (Enterprise Edition onyl)
 Marketplace URL: https://marketplace.owncloud.com/apps/ransomware_protection[Ransomware Protection]
 
 Use these commands to help users recover from a Ransomware attack. 
-You can find more information about the application in the documentation <../../enterprise/ransomware-protection/index>.
+You can find more information about the application in the documentation 
+<../../enterprise/ransomware-protection/index>.
```

Comment:
This looks like an rST-style link. It doesn't render properly with Antora.
![bildschirmfoto 2018-10-01 um 12 02 12](https://user-images.githubusercontent.com/196801/46282732-94e85d80-c572-11e8-83a4-2bc3c9a019ca.png)

Reply:
Completely right, was originaly in the doc, I did not change it.
Remember the chat on talk, we have to fix more than this link. You took the task on the example discussed how to do it and I will take that example as template for other fixes 😄

Reply:
Sorry, I don't quite understand "You took the task on the example discussed how to do it".

Reply:
talk:
**me:**
proposal:
I will remove the false label in occ_app_commands.
mind that you fix the correct linking to `[[password-policy]` ?
(and I will learn how you did it 🙂 )
**you**
sure
**me**
info, this will help, as in the occ_apps_commands file, at section 
`[[ransomware-protection]]`, we have a "link" that is rendered as text that needs correction `<../../enterprise/ransomware-protection/index>`.
btw, just seen as we chat...